plugins/render/api_key.go

```diff
@@ -0,0 +1,88 @@
+package render
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	Profiles map[string]Profile
+}
+
+type Profile struct {
+	APIKey string `yaml:"apiKey"`
+}
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://render.com/docs"),
+		ManagementURL: sdk.URL("https://dashboard.render.com/u/settings#api-keys"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.APIKey,
+				MarkdownDescription: "API Key used to authenticate to Render.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 30,
+					Charset: schema.Charset{
+						Uppercase: true,
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+		},
+		DefaultProvisioner: provision.TempFile(renderConfig, provision.AtFixedPath(ConfigPath())),
+		Importer: importer.TryAll(
+			TryRenderConfigFile(),
+		),
+	}
```

Comment:
No need for `importer.TryAll`, if we only import from one source. You can directly use `TryRenderConfigFile()`.

Reply:
resolved